changelog/21641.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+auto-auth: support ldap auth
```

Comment:
@VioletHynes or @marcboudreau would one of you be able to update this file to use the right format for a new feature (see the changelog process page) or change the section to an improvement?

Reply:
I'll get this fixed shortly :)